commands/createfixpullrequests_test.go

```diff
@@ -155,12 +155,6 @@ func TestFixPackageVersion(t *testing.T) {
 				// Verify that case-sensitive packages in python are lowered
 				assert.Contains(t, string(file), strings.ToLower(test.impactedPackaged))
 			})
-			t.Run(test.technology.ToString(), func(t *testing.T) {
```

Comment:
Removed these tests as we will write new tests that will address every indirect fix we will implement